db/test/test_stats.py

```diff
@@ -296,14 +296,27 @@ def add_empty_lowlevel(mbid, lossless, date):
     gid_type = gid_types.GID_TYPE_MSID
     with db.engine.connect() as connection:
         query = text("""
-            INSERT INTO lowlevel (gid, build_sha1, lossless, submitted, gid_type)
```

Comment:
I can't remember why we made our own insert method here. As this tests the stats module and not data, could we use the db.data module instead of duplicating code here?

Reply:
The functions that would be used here from the db.data module are nested within `write_low_level`, so they aren't accessible. Should I write the function to calculate next submission offset outside of `write_low_level` so that it can be used in other places like here, or is it best to leave this duplicated?

Reply:
OK, revisiting my comment from https://github.com/metabrainz/acousticbrainz-server/pull/340#discussion_r285563569 - we have this code twice in tests now. Let's move the nested function to the top level of the data module so that we can use it here and in test_data.